resources/TrashPandaTheme.icls

```diff
@@ -586,24 +578,19 @@
         <option name="FOREGROUND" value="6a7175" />
       </value>
     </option>
-    <option name="REGEXP.BRACES">
```

Comment:
It looks like the regex updates that were updated in the last release are being modified here. I didn't start up PyCharm but these changes are affecting the "RegExp" color scheme settings generally.